package.json

```diff
@@ -175,6 +175,11 @@
     "wireit": "0.13.0",
     "zod": "3.22.2"
   },
+  "overrides": {
+    "@microsoft/api-extractor": {
+      "typescript": "$typescript"
```

Comment:
Why do we need to override the `@microsoft/api-extractor`'s TS ?

Reply:
Because it uses an older version which is incompatible.